docs/tutorials/dotnet-systemd.md

```diff
@@ -1,74 +1,88 @@
-# Run a .Net Echo Bot as a systemd service on Ubuntu WSL
+# Run a .NET Echo Bot as a systemd service on Ubuntu WSL
 *Authored by Oliver Smith ([oliver.smith@canonical.com](mailto:oliver.smith@canonical.com))*
 
-.Net is an open-source development platform from Microsoft that enables developers to build multi-platform applications from a single codebase.
+In this tutorial we will take advantage of WSL's systemd support to run a chatbot as a systemd service for easier deployment.
 
-In this tutorial we demonstrate how easy it is to start working with .Net on Ubuntu WSL by creating a simple chatbot accessible from your Windows host. We also take advantage of WSL's new systemd support to run our bot as a systemd service for easier deployment.
+We will create the bot using .NET on Ubuntu WSL and it will be accessible from the Windows host.
+
+.NET is an open-source development platform from Microsoft that enables developers to build multi-platform applications from a single codebase.
 
 ## Requirements
 
 * A PC running Windows 11
 * The latest version of the WSL Windows Store application
-* Ubuntu, Ubuntu 22.04 LTS or Ubuntu Preview installed
+* Ubuntu 22.04 LTS, Ubuntu 24.04 LTS or Ubuntu Preview installed
+
+Systemd support is a new feature of WSL and only available on WSL version 0.67.6 or higher.
 
-Systemd support is a new feature of WSL and only available on WSL version XX or higher.
+In your PowerShell terminal, you can check your current WSL version by running:
 
-You can check your current WSL version by running:
+> `wsl --version`
 
-> `> wsl --version`
+Inside WSL, you can check that systemd is enabled on your Ubuntu distribution by running:
 
-In your PowerShell terminal.
+> `cat /etc/wsl.conf`
 
-To enable systemd on your Ubuntu distribution you need to add the following content to `/etc/wsl.conf`​:
+If enabled the output will be:
 
 > `[boot]`  
 > `systemd=true`
 
+If systemd is set to `false` then open the file with `sudo nano /etc/wsl.conf`, set it to `true` and save.
 Make sure to restart your distribution after you have made this change.
 
-## Install .Net
+## Install .NET
 
-.Net has recently [been added to the Ubuntu repositories](https://ubuntu.com/blog/install-dotnet-on-ubuntu). This means you can now quickly install a bundle with both the SDK and runtime with a single command:
+```{note}
+In Ubuntu 24.04 LTS .NET 8 is available [in the Ubuntu repositories](https://ubuntu.com/blog/install-dotnet-on-ubuntu).
+To install this version of .NET for your project you only need to run:
 
->`$ sudo apt install dotnet6`
+> `sudo apt install dotnet8`
 
-Run `dotnet --version` to confirm that the package was installed successfully.
+In this tutorial we will be using .NET 6, which requires adding the backports archive for older .NET versions on Ubuntu 24.04 LTS.
+If you are using Ubuntu 22.04 LTS you can skip this step.
```

Comment:
This is confusing. We talk about .NET 8 in previous paragraph, then say that we are using 6 even on 24.04. So why talk about 8 at all? Is this tutorial invalid on 8?

Reply:
The original tutorial instructed to download dotnet with `sudo apt install dotnet6`.
.NET8 is available in 24.04 but the bot template used in the tutorial is only available for .NET6.
Users with 22.04 or 24.04 installed can both complete the tutorial with .NET6 but 24.04 requires the backport archive step.

I agree that mention of .NET8 is confusing here but I was trying to indicate that the latest version of the framework is available on the latest LTS.

For the sake of ensuring that this tutorial **works** for now I think we should advise using .NET6 and remove reference to .NET8.